libs/payments/paypal/src/lib/client.ts

```diff
@@ -122,8 +121,49 @@ export class PayPalClient {
       return resultObj;
     } else {
       // TypeScript doesn't narrow ACK, necessitating a cast
-      throw new PayPalClientError(result.text, resultObj as NVPErrorResponse);
+      throw new MultiError(
+        this.getListOfPayPalClientErrors(
+          result.text,
+          resultObj as NVPErrorResponse
+        )
+      );
+    }
+  }
+
+  public getListOfPayPalClientErrors(raw: string, data: NVPErrorResponse) {
```

Comment:
I opted to leave this helper in this file, as this is the only place it's used. Happy to move it somewhere more appropriate if preferred.

Reply:
I don't see it using anything in the class instance, so it should be either a static method, or it could be moved outside the class in this file.